front/src/modules/trainschedule/components/Timetable/PacedTrain/OccurrenceItem.tsx

```diff
@@ -201,25 +206,28 @@ const OccurrenceItem = ({
         )}
       </div>
 
-      {isException(occurrence) && occurrence.isValid && !disabled && (
-        <div className="more-info">
-          <div className="more-info-left">
-            {/* TODO : add a category span in https://github.com/OpenRailAssociation/osrd/issues/11542 */}
-            <span className="more-info-item">
-              {t('timetable.stopsCount', { count: occurrence.stopsCount })}
-            </span>
-            <span className="more-info-item">{occurrence.pathLength}</span>
-            <span className="more-info-item m-0" data-testid="allowance-energy-consumed">
-              {occurrence.mechanicalEnergyConsumed}&nbsp;kWh
-            </span>
-          </div>
-          {occurrence.duration && (
-            <div className="duration-time">
-              <span data-testid="train-duration">{formatTrainDuration(occurrence.duration)}</span>
+      {isException(occurrence) &&
+        occurrence.isValid &&
+        !disabled &&
+        isExceptionFromPathOrSimulation(occurrence) && (
```

Comment:
```suggestion
      {
        occurrence.isValid &&
        !disabled &&
        isExceptionFromPathOrSimulation(occurrence) && (
```
If i'm not mistaken, when `isExceptionFromPathOrSimulation() === true` it means that we're dealing with an exception, so I think the 1st check isn't necessary :)

Reply:
Yes but it can be false if it is an exception by name

Reply:
Agreed with @SarahBellaha, the first test is not necessary

Reply:
You're right!